src/components/network/network.js

```diff
@@ -152,6 +152,10 @@ genotet.NetworkView = function(viewName, params) {
     .on('genotet.nodeClick', function(event, node) {
       this.panel.displayNodeInfo(node);
       this.loader.incidentEdges(node);
+      this.signal('link', {
```

Comment:
Anyway, I think they are the same type of event and can be listened by the same listener.